### PR TITLE
More pythonic code

### PR DESCRIPTION
**Pull request by marcu**
I propose a improvement of the python code : instead of using a loop `for` with a `break`, we can use the method `next`.

### CODE REVIEW

File: docs/pyqgis_developer_cookbook/vector.rst [hunk 1]
```diff
@@ -215,9 +215,9 @@ can call :meth:`select() <qgis.core.QgsVectorLayer.select>` passing to it the li
  selected_fid = []
 
  # Get the first feature id from the layer
- for feature in layer.getFeatures():
+ feature = next(layer.getFeatures())
+ if feature :
```
Comment (DelazJ) [inline docs/pyqgis_developer_cookbook/vector.rst:219]: nitpicking?
```suggestion
 if feature:
```

File: docs/pyqgis_developer_cookbook/vector.rst [hunk 2]
```diff
@@ -215,9 +215,9 @@ can call :meth:`select() <qgis.core.QgsVectorLayer.select>` passing to it the li
...
      selected_fid.append(feature.id())
-     break
 
  # Add these features to the selected list
```
Comment (DelazJ) [inline docs/pyqgis_developer_cookbook/vector.rst:222]: This should be singular, right?
```suggestion
 # Add that feature to the selected list
```

**Review by DelazJ — COMMENTED** [review]
Thanks @marcu

File: docs/pyqgis_developer_cookbook/vector.rst (hunk 1 above)
Comment (marcu) [reply docs/pyqgis_developer_cookbook/vector.rst:219]: Ha ha. No you are right !

File: docs/pyqgis_developer_cookbook/vector.rst (hunk 2 above)
Comment (marcu) [reply docs/pyqgis_developer_cookbook/vector.rst:222]: Thanks !

**Pull request merged**